packages/config/config/devices/0x019b/4512688.json

```diff
@@ -1,5 +1,5 @@
 {
-	"manufacturer": "Heatit",
+	"manufacturer": "Heatit Controls AB",
```

Comment:
The way we use this field, its more like the brand a device is being marketed as, not the actual manufacturer.
AFAIK these devices are branded "Heatit", no?

Reply:
Yes thats correct, this has been updated